benchmark/clara.hpp

```diff
@@ -0,0 +1,1260 @@
+// Copyright 2017 Two Blue Cubes Ltd. All rights reserved.
```

Comment:
Move this to the external folder create folder called clara

Reply:
Done